pkg/pool-weighted/contracts/managed/ManagedPool.sol

```diff
@@ -116,33 +108,41 @@ contract ManagedPool is ManagedPoolSettings {
         SwapRequest memory request,
         uint256 balanceTokenIn,
         uint256 balanceTokenOut,
-        bytes32 poolState,
-        bytes32 tokenInState,
-        bytes32 tokenOutState
-    ) internal view returns (uint256) {
-        uint256 weightChangeProgress = ManagedPoolStorageLib.getGradualWeightChangeProgress(poolState);
-
-        // TODO: Once all weights are normalized, change to
-        // uint256 tokenInWeight = ManagedPoolTokenLib.getTokenWeight(tokenInState, weightChangeProgress)
-        // uint256 tokenOutWeight = ManagedPoolTokenLib.getTokenWeight(tokenOutState, weightChangeProgress)
-        uint256 tokenInWeight = _getNormalizedWeight(request.tokenIn, weightChangeProgress);
-        uint256 tokenOutWeight = _getNormalizedWeight(request.tokenOut, weightChangeProgress);
-
-        uint256 scalingFactorTokenIn = ManagedPoolTokenLib.getTokenScalingFactor(tokenInState);
-        uint256 scalingFactorTokenOut = ManagedPoolTokenLib.getTokenScalingFactor(tokenOutState);
+        bytes32 poolState
+    ) internal view returns (uint256 swapOutput) {
```

Comment:
We've called this 'amount calculated' (as opposed to amount given) in the past - not sure if we want to reproduce that here as it's a bit too general maybe. 'output' might be confusing in that it could be an amount in.